src/components/UI/PlioThumbnail.vue

```diff
@@ -1,13 +1,13 @@
 <template>
   <!-- named routes - https://router.vuejs.org/guide/essentials/named-routes.html -->
-  <router-link :to="{ name: 'Player', params: { id: plio.plio_id } }" tag="div" class="plio">
+  <router-link :to="{ name: 'Player', params: { id: plio.uuid } }" tag="div" class="plio">
```

Comment:
TODO - routing to the player using the `uuid` instead of the `id`

Reply:
is this still a TODO?

Reply:
Yes
Right now it's just passing the uuid as a prop to the player component
The component itself needs to be configured with this prop